src/translations/indexMapper.js

```diff
@@ -0,0 +1,362 @@
+import { arrayFilter, arrayMap } from './../helpers/array';
+import { getListWithRemovedItems, getListWithInsertedItems } from './maps/utils/visuallyIndexed';
+import { rangeEach } from '../helpers/number';
+import IndexMap from './maps/indexMap';
+import SkipMap from './maps/skipMap';
+import MapCollection from './mapCollection';
+import localHooks from '../mixins/localHooks';
+import { mixin } from '../helpers/object';
+import { isDefined } from '../helpers/mixed';
+
+class IndexMapper {
+  constructor() {
+    /**
+     * Map storing the sequence of indexes.
```

Comment:
Inconsistency: This explanation is different than in `indexMap.js`.

Reply:
For me it's ok. We will discuss that.

Reply:
For discussion.